stripe-connect/src/main/java/com/stripe/android/connectsdk/PrivateBetaConnectSDK.kt

```diff
@@ -0,0 +1,11 @@
+package com.stripe.android.connectsdk
+
+/**
+ * Marks an API for Private Beta usage, meaning it can be changed or removed at any time (use at your own risk).
+ * If you are interested in using a feature marked for private beta, send an email to
+ * [private-beta-element@stripe.com](mailto:private-beta-element@stripe.com).
+ */
+@RequiresOptIn(message = "This API is under construction. If you're interested in using it, email private-beta-element@stripe.com.")
```

Comment:
Can you make the javadoc and the message here specific to the connect SDK private beta? A good example of this is the [ExperimentalCustomerSessionApi](https://github.com/stripe/stripe-android/blob/186bbb5f5702df7e8fd5016c99174681b8590441/paymentsheet/src/main/java/com/stripe/android/paymentsheet/ExperimentalCustomerSessionApi.kt#L10) annotation. (You could also get rid of the javadoc entirely if you want, I think)